File: t/CS/LTDMergeDemo/WizardControl.cs
// *************************************************************
// Copyright (c) 1991-2019 LEAD Technologies, Inc.              
// All Rights Reserved.                                         
// *************************************************************
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace LTDMergeDemo
{
   public partial class WizardControl : TabControl
   {
      public WizardControl()
      {
         InitializeComponent();
      }

      protected override void WndProc(ref Message m)
      {
         const int TCM_ADJUSTRECT = 0x1328;
         if(!DesignMode && m.Msg == TCM_ADJUSTRECT)
         {
            m.Result = (IntPtr)1;
         }
         else
         {
            base.WndProc(ref m);
         }
      }
   }
}

File: t/VB/Ocr2SharePoint2010Demo/WizardControl.vb
' *************************************************************
' Copyright (c) 1991-2019 LEAD Technologies, Inc.              
' All Rights Reserved.                                         
' *************************************************************
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace Ocr2SharePointDemo
   Public Partial Class WizardControl : Inherits TabControl
	  Public Sub New()
		 InitializeComponent()
	  End Sub

	  Protected Overrides Sub WndProc(ByRef m As Message)
		 Const TCM_ADJUSTRECT As Integer = &H1328
		 If (Not DesignMode) AndAlso m.Msg = TCM_ADJUSTRECT Then
			m.Result = CType(1, IntPtr)
		 Else
			MyBase.WndProc(m)
		 End If
	  End Sub
   End Class
End Namespace
